torchtune/models/code_llama2/__init__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from ._model_builders import (  # noqa
+    code_llama2_13b,
+    code_llama2_70b,
+    code_llama2_7b,
+    lora_code_llama2_13b,
+    lora_code_llama2_70b,
+    lora_code_llama2_7b,
+    qlora_code_llama2_13b,
+    qlora_code_llama2_70b,
```

Comment:
I don't know that we should add any QLoRA + 70B models yet, since it will require FSDP. And as of right now the combination of these two is not yet well-supported. Cc @rohan-varma for thoughts here